src/store/transaction/TransactionActions.js

```diff
@@ -49,42 +51,69 @@ export class TransactionActions {
   }
 
   @ThunkAction
-  prepareTransaction(id, from, to, amount) {
+  prepareTransaction(id, from, to, amount, platform) {
     return async dispatch => {
-      const toId = await this.accountResolver.getIdByAccountName(to);
-      const accountName = await this.accountResolver.getAccountNameById(toId);
-      if (accountName !== to) {
-        throw new Error('Can\'t resolve "to" account');
-      }
+      let transaction;
+
       const currentBalance = await this.privateSnaxProvider.getBalance(from);
-      const transaction = {
-        balance: currentBalance,
-        id,
-        from,
-        to: { platformId: toId, platformAccountName: to },
-        amount
-      };
-      dispatch(
-        this.setTransactionToSign(
-          currentBalance,
+
+      if (platform === 'twitter') {
+        const toId = await this.accountResolver.getIdByAccountName(to);
+        const accountName = await this.accountResolver.getAccountNameById(toId);
+
+        if (accountName !== to) {
```

Comment:
We need to check at least one of them to be not null because if they are both nulls, check will be passed but this behaviour is incorrect

Reply:
I believe both of them should exist and not be equal to null. Any case why we will need a null as a value for one field and not nullable for the second one?

Reply:
I mean null === null check will bypass this exception block and we will try to transfer tokens to null